### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by gquemener**

### CODE REVIEW

**Comment by gquemener** [comment]
I wanted to fix `therefor` => `therefore` everywhere, but I've discovered that it exists without an "e" at the end (thanks!): https://www.grammarly.com/blog/therefore-vs-therefor/

> Therefor, you should start with a single module 

Can be replaced by "for that", without losing the sense.

> They avoid accidental complexity by ensuring context boundaries and therefor a clear focus on a specific problem.

The sentence loses its meaning when replacing "therefor" by "for that". As it makes sense to replace it with "as a consequence", I would write "therefore".

Do you agree?

File: docs/intro/module_system.html [hunk 1]
```diff
@@ -644,7 +644,7 @@ <h2 id="1-3-2">Application Layer</h2>
 <p>The application layer uses the <code>App</code> namespace.</p>
 <h2 id="1-3-3">Modules</h2>
 <p>A module has its own namespace. For example the <em>RealtyRegistration</em> module uses the namespace <code>FeeOffice\RealtyRegistration</code>.
-The namespace maps to the path <code>src/RealtyRegistration/src</code>.</p>
+The namespace maps to the path <code>src/RealtyRegistration</code>.</p>
```
Comment (codeliner) [inline docs/intro/module_system.html:647]: It's not a typo ;). Each module is itself an "app" with src and tests folder, see here: https://github.com/proophsoftware/fee-office/tree/master/src/RealtyRegistration/src

And here is the autoloading: https://github.com/proophsoftware/fee-office/blob/master/src/RealtyRegistration/composer.json#L35

**Comment by gquemener** [comment]
I close it as the change does not fix anything!

Question in https://github.com/proophsoftware/fee-office/pull/1#issuecomment-432335375 is still pending though ;)

I'm following this case study closely, well done!

**Comment by codeliner** [comment]
oh sorry, didn't see the question. Yeah, sounds good. Wording correction/enhancement is always welcome. I try to improve my English writing skills, but it's not easy ;)

**Pull request closed without merging**